### DISCUSSION TITLE
Dashboard : Settings : MFA

### DISCUSSION THREAD

**Discussion by john-tmc** [planetscale/discussion#482] [Feature Requests]
Hi! I know this is just a minor request, should be easy to implement 👍 

Request :
-   Have the TOTP code below / toggle to show TOTP code, and a copy button. 

Case :
When enabling multi-factor authentication, I am unable to see the TOTP code, I am using desktop and Bitwarden doesn't detect the image, so I usually copy & paste the TOTP code instead (same could be said for those with a broken camera phone).

Thanks!

**Comment by orware** [comment]
Hello @john-tmc,

Is the issue you are running into something like the following where the QR code isn't being displayed when going to enable Multi-factor authentication?
![image](https://github.com/planetscale/discussion/assets/931650/e43ea3c5-5dab-4878-b641-66cc03e4984b)

I did notice if I made my browser window small enough that the QR code would start to be hidden more and more so I'm wondering if this is the same issue you are describing or if it's a bit different?
